bin/rucio

```diff
@@ -384,75 +458,169 @@ def list_file_replicas(args):
         print('The substitution parameter must equal --link="/pfn/dir:/dst/dir"')
         return FAILURE
 
-    for did in args.dids:
-        scope, name = get_scope(did, client)
-        client.get_metadata(scope=scope, name=name)  # break with Exception before streaming replicas if DID does not exist
-        dids.append({'scope': scope, 'name': name})
-
-    replicas = client.list_replicas(dids, schemes=protocols,
-                                    ignore_availability=True,
-                                    all_states=args.all_states,
-                                    rse_expression=args.rses,
-                                    metalink=args.metalink,
-                                    client_location=detect_client_location(),
-                                    sort=args.sort, domain=args.domain,
-                                    resolve_archives=not args.no_resolve_archives)
-    rses = [rse["rse"] for rse in client.list_rses(rse_expression=args.rses)]
-
-    if args.metalink:
-        print(replicas[:-1])  # last character is newline, no need to print that
-        return SUCCESS
+    if cli_config == 'rich':
+        status = Status('Fetching file replicas', spinner=CLITheme.SPINNER, spinner_style=CLITheme.SPINNER_STYLE, console=console)
+        with status:
```

Comment:
[See this comment](https://github.com/rucio/rucio/pull/7012/files#r1763387604) - I won't replicate it for each part of the code where it applies but keep it in mind please.